src/components/icon-tile/types.ts

```diff
@@ -8,7 +8,7 @@ import { ProductSlug } from 'types/products'
 export type ProductBrandColor =
 	| 'neutral'
 	| 'neutral-dark'
-	| Exclude<ProductSlug, 'sentinel' | 'hcp'>
+	| Exclude<ProductSlug, 'hcp'>
```

Comment:
I don't think this change is necessary since as far as I know Sentinel does not have a brand color

Reply:
```suggestion
	| Exclude<ProductSlug, 'sentinel' | 'hcp'>
```